### PR TITLE
Latest upstream 7.2.24

### PR DESCRIPTION
**Pull request by sethryder**

### CODE REVIEW

**Review by carlwgeorge — APPROVED** [review]

File: php72u.spec [hunk 1]
```diff
@@ -1792,6 +1792,9 @@ exit 0
 
 
 %changelog
+* Mon Oct 24 2019 Seth Ryder <seth@sethryder.com> - 7.2.24-1
+- Re-add pid file used by logrotate to fix logrotate issue
```
Comment (carlwgeorge) [inline php72u.spec]: Can you adjust this changelog entry to say it's updating to the latest upstream version?

**Review by carlwgeorge — CHANGES_REQUESTED** [review]

File: php72u.spec (hunk 1 above)
Comment (sethryder) [reply php72u.spec]: sorry about that. just pushed the updated changelog entry.

Comment (carlwgeorge) [reply php72u.spec]: No worries, that's what reviews are for.  :grin:

**Review by carlwgeorge — APPROVED** [review]

**Comment by carlwgeorge** [comment]
Thanks Seth. php72u-7.2.24-1.el7 has been published to the testing repo.

**Pull request merged**